pkg/actuators/machine/reconciler.go

```diff
@@ -251,3 +427,7 @@ func (r *Reconciler) setMachineCloudProviderSpecifics(instance *vpcv1.Instance)
 	r.machine.Labels[machinecontroller.MachineInstanceTypeLabelName] = r.providerSpec.Profile
 
 }
+
+func stringPointerRef(value string) *string {
```

Comment:
could we use the k8s utility pointer package instead?
https://pkg.go.dev/k8s.io/utils/pointer#String

Reply:
ohh, yeah, I can make that change

Reply:
updated